Safety/Safety-Systems/AuxTel-Safety-Gate-Procedures.rst

```diff
@@ -0,0 +1,198 @@
+    and the telescope and dome drives will remain disabled. 
+
+    If :guilabel:`Release` is pressed the AuxTel IS will register the gate as open, even if the gate is physically closed.
+
+.. Fig3. Gate fully opened 
+
+.. figure:: ./_static/gate-fully-open.png
+      :align: left
+      :width: 335px
+      
+      Gate fully opened
+
+.. Fig4. Gate in closed position
+
+.. figure:: ./_static/gate-fully-closed.png
+      :align: right
+      :width: 335px
+      
+      Gate in closed position
+      
+                                       
+.. _AuxTel-Safety-Gate-Procedures-Gate-operation-Opening-the-gate:
+
+Opening the gate 
+------------------
+
+.. - section 4
+
+When the gate is fully closed, to open it:
+
+1. Press the :guilabel:`Release` button on the gate. 
+    The blue *Lock/Area Safe* indicator will turn off.
+
+2. Move the **handle** and physically move the gate to the open position.
+
+    .. Fig6. Lock button                                                
+
+    .. figure:: ./_static/lock-release-buttons.png
+        :align: left
+        :width: 250px
+        
+        :guilabel:`Release` and :guilabel:`Lock` buttons
+
+
+.. _AuxTel-Safety-Gate-Procedures-Gate-operation-Closing-the-gate:
+
+Closing the gate
+------------------
+.. section 3
+
+When the gate is fully open, to close it:
+
+1. Move the gate to the **closed position**.
+
+2. Insert the handle into the frame. 
+    Notice that the *Safety Gate* indicator lights up.
+
+    .. figure:: ./_static/handle-in-closed-position.png
+      :width: 250px
+      
+      Handle in closed position
+
+    .. Fig5. Handle in closed position
+
+3. Press the :guilabel:`Lock` button. 
+    You should hear a click, and the *Lock/Area Safe* indicator should illuminate in blue, signaling that the handle cannot be moved and the gate is securely closed. 
+
+    .. Fig7. Locked gate indicator
+
+    .. figure:: ./_static/locked-gate-indicator.png
+        :width: 250px
+        
+        Locked gate indicator
+
+.. _AuxTel-Safety-Gate-Procedures-Verify-Telescope-and-Dome-Drive-Interlocks:
+
+Verify Telescope and Dome Drive Interlocks 
+===========================================
+
+.. section 8
+
+To determine if the drives have a **triggered interlock**, due to the gate open, gate in Error, E-STOP action, or general failure. 
+Do the following:
+
+* Check the **Pilz controller** inside the AuxTel Main Cabinet.
+
+* According to the **lights** if they are:
+    * Off, an interlock is triggered.
+    * Illuminated, the system is functioning correctly.
+
+.. Fig 11. Pilz controller status
+
+.. figure:: ./_static/pilz-controller-status.png
+      :width: 250px
+      
+      Pilz controller status
+
+
+
+.. _AuxTel-Safety-Gate-Procedures-Enable-the-gate-after-emergency-release:
+
+Enable the gate after emergency release
+========================================
+
+.. section 5
+
+1. **Rearm the red handle** and move it to its normal position, as shown in figure below. 
```

Comment:
I would add: Rearm the handle "manually"... just to clarify that there is no need to press any other button or that it is not an automatic process.
It might be a good idea to have another picture with the red handle pressed showing the mechanism opened/activated. So you can show the before->after. The description of the full procedure to rearm the gate could be too much.